assimilation_code/modules/utilities/utilities_mod.f90

```diff
@@ -688,7 +688,7 @@ subroutine error_handler(level, routine, text, src, rev, rdate, aut, text2, text
 ! add trailing blanks here because trim will strip them below.
 
 wherefrom = trim(taskstr)//' '//trim(routine)
-wherecont = trim(taskstr)//' '//trim(routine)//' ...'
+wherecont = trim(taskstr)//' '//trim(routine)//'...'
```

Comment:
oh bother ...  I don't know why there is no space before the ellipses ... but should be reverted to avoid nuisance changes
```suggestion
wherecont = trim(taskstr)//' '//trim(routine)//' ...'
```